src/main/java/com/missafternoon/budjen/transactions/Validator.java

```diff
@@ -10,16 +10,16 @@ public boolean validate(String[] args) {
                 && contains(args, "--description")
                 && contains(args, "--amount")
                 && (contains(args, "--credit") ^ contains(args, "--debit"))
-        )) {
+            )) {
```

Comment:
Possibly the indentation discrepancy? To be honest, this is just a guess!

Reply:
Nope, sorry. This was fixing a checkstyle violation: The indentation of `)` didn't match the indentation of `(`.